.github/workflows/nightly-merge.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Nightly Merge (master to iiot)'
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  nightly-merge:
```

Comment:
From https://github.com/marketplace/actions/nightly-merge, it mentions that "if the merge fails due to conflicts, the action will fail, and the repository maintainer should perform the merge manually".  @varunpuranik do you know who are our repo maintainer?

Reply:
FYR https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization